### ISSUE TITLE
Accesskit integration

### ISSUE THREAD

**Issue by matthunz** [concoct-rs/viewbuilder#7]
The plan is to use `accesskit` for accessibility support. This should be fairly straightforward by adding the `winit` adapter and an `Element::semantics` or `accessibility` method.
